elements/timeline-view/timeline-toast.html

```diff
@@ -137,13 +203,16 @@
             this.$.getPresets.url = "/benine/cameras/" + this.camera.id + "/presets";
             this.$.getPresets.generateRequest();
             this.$.toast.open();
+
+            this.$.instruments_available.hidden = this.block_data.instruments.size <= 0;
+            this.$.instruments_unavailable.hidden = this.block_data.instruments.size > 0;
```

Comment:
Sure. There's plenty of places I can did this, so I'll run through my changes and do it where I can.